db/migrate/20210309100557_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.string :password
+      t.boolean :admin
```

Comment:
@josephelias-rubyians  Its a good practice to provide default value to boolean datatypes:
If default not given, then app needs to deal with 3 values - true, false & null. We often miss to do the null checking and eventually breaks the code.
If default given, then app can deal with only 2 values - true & false.

Please provide default as false in this usecase.